base/strings/io.jl

```diff
@@ -134,14 +134,15 @@ function _str_sizehint(x)
     end
 end
 
+function _str_sizehints(xs...)
+    mapreduce(_str_sizehint, +, xs; init = 0)
```

Comment:
```suggestion
    sum(_str_sizehint, xs; init = 0)
```

Reply:
This whole function is so simple that you might as well just call `sum` twice below, and delete the `_str_sizehints` function, IMO.